### ISSUE TITLE
Cannot find module stopwatch-node/dist/stopwatch

### ISSUE THREAD

**Issue by chrisbbreuer** [permafrost-dev/node-ray#176]
**Describe the bug**

I am experiencing an issue where an upgrade from `v1.20.7` to `v1.20.11` is throwing the following error:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module './stacks/node_modules/.pnpm/node-ray@1.20.11/node_modules/stopwatch-node/dist/stopwatch' imported from /.stacks/node_modules/.pnpm/node-ray@1.20.11/node_modules/node-ray/dist/index.esm.mjs
Did you mean to import stopwatch-node@1.1.0/node_modules/stopwatch-node/dist/stopwatch.js?
```

_Please note, I haven't tested .8, .9 or .10 versions._

The relating `node_modules` folder looks as follows: 

<img width="340" alt="image" src="https://github.com/permafrost-dev/node-ray/assets/6228425/c65f72b8-c200-4a0b-b2a1-1619c679326b">

**Expected behavior**

From the little investigating I have done is that it seems relating this import: 

[`import * as StopWatches from 'stopwatch-node/dist/stopwatch'`](https://github.com/permafrost-dev/node-ray/blob/cfd596ad91d14c0566bfeb62acee4a1f82430087/src/Stopwatch/Stopwatch.ts#L3)

Unsure still what the reason is as to how this is being caused. Maybe something speaks out to you 🙂

**Environment (please complete the following information):**

 - OS: macOS Ventura 13.3.1
 - NodeJS Version: 18.16.0
 - Language & Version: Typescript 5.0.4

**Additional context**

**Comment by patinthehat** [comment]
@chrisbbreuer Thank you for the bug report, I am looking into this now and will post an update shortly.

**Comment by patinthehat** [comment]
@chrisbbreuer I've published version `1.21.0` which should resolve this issue.  Please open a new issue if you experience this bug after updating. Thanks again! :+1:
